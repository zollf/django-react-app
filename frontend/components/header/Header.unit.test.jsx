import React from 'react';
import { render } from '@testing-library/react';

import Header from './index.jsx';

it('matches its snapshot', () => {
  expect(render(<Header />).asFragment()).toMatchSnapshot();
});
