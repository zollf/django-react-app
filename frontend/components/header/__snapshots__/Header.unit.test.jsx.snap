// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches its snapshot 1`] = `
<DocumentFragment>
  <div>
    This is header?
  </div>
</DocumentFragment>
`;
